crates/store/re_grpc_server/src/lib.rs

```diff
@@ -294,15 +307,92 @@ pub fn spawn_with_recv(
             }
         }
     });
-    channel_rx
+    tokio::spawn(async move {
+        loop {
+            let msg = match broadcast_table_rx.recv().await {
+                Ok(msg) => msg.data.decode().map(|data| TableMsg {
+                    id: msg.id.into(),
+                    data,
+                }),
+                Err(broadcast::error::RecvError::Closed) => {
+                    re_log::debug!("message proxy server shut down, closing receiver");
+                    // We don't have to explicitly call `quit` on crossbeam channels.
+                    break;
+                }
+                Err(broadcast::error::RecvError::Lagged(n)) => {
+                    re_log::debug!(
+                        "message proxy receiver dropped {n} messages due to backpressure"
+                    );
+                    continue;
+                }
+            };
+            match msg {
+                Ok(msg) => {
+                    if channel_table_tx.send(msg).is_err() {
+                        re_log::debug!(
+                            "message proxy smart channel receiver closed, closing sender"
+                        );
+                        break;
+                    }
+                }
+                Err(err) => {
+                    re_log::error!("dropping table due to failed decode: {err}");
```

Comment:
For consistency, use upper case initials in your log messages. Remember that INFO, WARN and ERROR are all shown to the user, both in the viewer and in the console.

```suggestion
                    re_log::error!("Dropping table due to failed decode: {err}");
```

Reply:
https://github.com/rerun-io/rerun/issues/9516